homeassistant/components/onvif/strings.json

```diff
@@ -71,6 +71,19 @@
       }
     }
   },
+  "entity": {
+    "switch": {
+      "autofocus": {
+        "name": "Autofocus"
+      },
+      "ir_lamp": {
+        "name": "IR Lamp"
```

Comment:
Should this be `IR lamp` (like before)?

> Device, Area, and Entity names all start with a capital letter, the rest of the words are lower case (unless it's a word that represents a brand/name/abbreviation of course).

(from https://developers.home-assistant.io/blog/2022/07/10/entity_naming/)